user-guide/Troubleshooting/Known_issues/KI_Unable_to_open_trended_parameter_cards_via_Cube_advanced_search.md

```diff
@@ -0,0 +1,29 @@
+---
+uid: KI_Unable_to_open_trended_parameter_cards_via_Cube_advanced_search
+---
+
+# Unable to open trended parameter cards via Cube advanced search
+
+## Affected versions
+
+From DataMiner 10.4.0 [CU12]/10.5.0/10.5.3 onwards.
+
+## Cause
+
+To facilitate communicate between different modules, a serializable `DataObject` is used to create instances from a string, relying on reflection. Because of an issue introduced in DataMiner 10.4.0 [CU12]/10.5.0/10.5.3 (RN 42552), this reflection process is broken, preventing the correct `MethodInfo` from being located. As a result, trended parameter cards cannot be opened directly via navigation in Cube.
+
+## Fix
+
+No fix is available yet.
```

Comment:
Would "a fix is in the pipeline" not be better?
Why? one can assume that "No fix is available yet." means that the investigation is not started or the problem is not yet detected, while currently a fix is under QA review, which means the fix is almost ready to eb submitted.

Reply:
@HansSolos, we typically avoid sharing detailed information about the status of a fix. Instead, we update the known issue and move it to the "resolved" section once the fix is available. This approach helps us manage expectations and ensures that we only communicate fixes once they are confirmed and released.